docker-compose.yml

```diff
@@ -9,7 +9,7 @@ services:
     environment:
       - DB_USER="postgres"
       - DB_PASS="postgres"
-      - DB_NAME="blog2_db_1"
+      - DB_NAME="elephant_in_the_room_dev"
 
 
   adminer:
```

Comment:
use pgadmin: https://hub.docker.com/r/fenglc/pgadmin4/ instead of adminer

Reply:
[done](https://github.com/lambdaclass/elephant_in_the_room/pull/1/commits/11e547a7b7d0406ca84189adf9eec6bfa57bce74)